### DISCUSSION TITLE
Welcome to discussions!

### DISCUSSION THREAD

**Discussion by jaskarth** [Vineflower/vineflower#158] [Announcements]
## Welcome to the Quiltflower discussions page!
This is the place to ask questions and give feedback on Quiltflower, as well as ask about the codebase and how to contribute. If you wish to submit a bug report, we ask that you do so on the [issues page](https://github.com/QuiltMC/quiltflower/issues). If you would like to discuss in a more realtime format, feel free to join us on [the discord](https://discord.quiltmc.org/toolchain). We look forward to hearing from you!

**Comment by l-Luna** [comment]
cool cool
